code/firmware/rosco_m68k_v2.1/rosco_m68k_private.asm

```diff
@@ -20,12 +20,12 @@
 ; bit 14    : Requires larger system data area
 ; bit 15    : Snapshot version
   ifd HUGEROM
-RELEASE_VER     equ     $E0000220
+RELEASE_VER     equ     $E0000230
```

Comment:
Did you mean to bump this? 2.20 is still in DEV, AFAIK this change should be part of that eventual release...

Reply:
Very good point, I've reverted that.